### DISCUSSION TITLE
multiple payout schemes, same coin/node?

### DISCUSSION THREAD

**Discussion by mboyd1** [coinfoundry/miningcore#1428] [Q&A] [answered]
if I want to offer SOLO, PPLNS and PROP options for a particular coin, do I need 3 different daemons (and thus wallets) for each pool to manage the coins in the wallet? so the payouts/walltes/utxos don't get mixed up?

**Comment by oliverw** [comment, marked as answer]
> if I want to offer SOLO, PPLNS and PROP options for a particular coin, do I need 3 different daemons (and thus wallets) for each pool to manage the coins in the wallet? so the payouts/walltes/utxos don't get mixed up?

One daemon will do.

**Reply by mboyd1** [reply]
> One daemon will do.

do they need unique pool wallet addresses?

**Reply by oliverw** [reply]
No.
